dlpy/applications/darknet.py

```diff
@@ -167,18 +178,28 @@ def Darknet(conn, model_table='Darknet', n_classes=1000, act='leaky', n_channels
         that are larger than those sizes are cropped.
         Valid Values: 'none', 'unique', 'randomresized', 'resizethencrop'
         Default: 'unique'
+    random_mutation : string, optional
+        Specifies how to apply data augmentations/mutations to the data in the input layer.
+        Valid Values: 'none', 'random'
+        Default: 'NONE'
```

Comment:
you should remove the default part as there is no default now

Reply:
this is the same as many below

Reply:
+ these at the end are dlpydicts as I do some naming manipulations, you can check the source of DlpyDict